### PR TITLE
Rfa 01b updated disclosures

### PR DESCRIPTION
**Pull request by lauherkcambria**
#### Which User story does this relate to?
- https://osi-cwds.atlassian.net/browse/CALS-38

#### Brief feature description
- since api has been fixed, we can add in the rest of the disclosure cards. NOTE: https://osi-cwds.atlassian.net/browse/TPT3-62 is still in testing.

#### Does this PR block/effect other stories?
- no

#### Do any commands needs to be run in order to run the project with this PR?
-no 

#### Code coverage percent diff as a result of this PR?
-karma: 
78.38% Statements 
64.38% Branches 
 71.84% Functions
79.18% Lines 
-rspec: 91.28%

### CODE REVIEW

**Comment by ratneshraval** [comment]
@lauherkcambria Closing comments from #252, feature test needs to be complete.

File: app/javascript/rfa_forms/rfa01b_edit_view/crimeBackgroundAgainstCohabitant.js [hunk 1]
```diff
@@ -67,7 +62,7 @@ export default class CrimeBackgroundAgainstCohabitant extends React.Component {
                   index={index}
                   crime={crime}
                   idPrefix='crimeBackgroundAgainstCohabitant'
-                  clickClose={this.props.removeCard}
+                  clickClose={this.clickClose}
```
Comment (ratneshraval) [inline app/javascript/rfa_forms/rfa01b_edit_view/crimeBackgroundAgainstCohabitant.js]: This works but how is the index value sent to the `clickClose` method?

Comment (lauherkcambria) [reply app/javascript/rfa_forms/rfa01b_edit_view/crimeBackgroundAgainstCohabitant.js]: with the line      
` <span onClick={() => this.props.clickClose(this.props.index)} className='pull-right glyphicon glyphicon-remove' />` in the criminal fields component.

**Comment by ratneshraval** [comment]
Some design issues,
1. "view" link needs to be part of the header line "Disclosure Instructions"
2. Horizontal line should separate each disclosures.
3. Name component, SSN and some fields have different indentation in "Applicant or other adult" section

**Comment by lauherkcambria** [comment]
@ratneshraval i have addressed the css issues, and have made a clean up css rfa 01b chore in jira for anything we find that needs a css fix so that we can merge this into dev by wed. https://osi-cwds.atlassian.net/browse/CALS-4900

**Pull request merged**